OrbitGl/SamplingReport.cpp

```diff
@@ -4,48 +4,44 @@
 
 #include "SamplingReport.h"
 
-#include <utility>
```

Comment:
please add a short description to the commit, you renamed fields/variables, added nodiscard where appropriate. get thread report now returns a pointer, and you changed return type from string to std::string.

Reply:
I added the commit message in third commit (will squash 2nd and 3rd) to avoid force-update for now